### PR TITLE
relesae: finish 5.0.6 and minor template improvements

### PR DESCRIPTION
**Pull request by unknwon**
## Test plan

n/a

### CODE REVIEW

**Review by coury-clark — APPROVED** [review]

File: dev/release/templates/patch_release_issue_template.md [hunk 1]
```diff
@@ -13,21 +13,19 @@ Arguments:
 
 This release is scheduled for **$RELEASE_DATE**.
 
-> **Attention developers:** to get your commits in `main` included in this patch release add the `backport-$MAJOR.$MINOR` to the PR to `main`.
+> **Warning**: To get your commits in `main` included in this patch release, add the `backport-$MAJOR.$MINOR` to the PR to `main`.
```
Comment (keegancsmith) [inline dev/release/templates/patch_release_issue_template.md:16]: very very minor: the introduction of this comma doesn't seem correct to me from a pedantic english POV. But high school english was a long time ago for me, nor was I very good at it.

Comment (unknwon) [reply dev/release/templates/patch_release_issue_template.md:16]: Do you mean `**Warning**:` vs `**Warning:**`, or vs `**Warning** [...]` (no comma at all)?

If the former vs, that's because I think GitHub only recognizes `> **Warning**` and `> **Note**` without anything else surrounded by `**`s. 😁

Comment (keegancsmith) [reply dev/release/templates/patch_release_issue_template.md:16]: The comma (`,`) added just after the word release

Comment (unknwon) [reply dev/release/templates/patch_release_issue_template.md:16]: OHHHH... missed that totally 😂 OK, not a native speaker but it read kinda smooth to me? At least Grammarly doesn't complain about it.

I'd remove the comma if the sentence is concatenated the other way around, e.g.


> Add the `backport-$MAJOR.$MINOR` to get your commits ...

**Pull request merged**